specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/norealm.auth.cred.not.required.correct.user/request.rpt

```diff
@@ -0,0 +1,9 @@
+
+connect http://localhost:8000/auth/
+connected
+
+write method "GET"
+write header "Authorization" "Basic YWRtaW46YWRtaW4="
+write close
+
+read status "404" "Not Found"

```

Comment:
It would be better to have a positive test here.  In other words, make the assumption the resource is valid.  We should generally only respond with 401 or 200 in these tests.